taskfiles/LocalDev.yaml

```diff
@@ -0,0 +1,18 @@
+  version: '3'
+
+  tasks:   
+    start:
+      desc: deploy using skaffold to k8s using current kube context (run with -o interleaved for interactive output)
+      vars:
+        NFD_VERSION: v0.13.3
+      cmd: |
+        minikube --profile nfd-dev start
+        # this one will point docker cli to the same daemon as used by minikube nfd-dev cluster
+        # that way skaffold will build images in the correct local repo.
+        eval $(minikube -p nfd-dev docker-env)
+        kubectl apply -k "https://github.com/kubernetes-sigs/node-feature-discovery/deployment/overlays/default?ref={{.NFD_VERSION}}"
+        skaffold dev
```

Comment:
nit: maybe it will better to use --trigger=manual to avoid unneeded rebuilds?

Reply:
done.